src/ui/LogdataStorage.cpp

```diff
@@ -221,15 +221,21 @@ double LogdataStorage::getTimeDivisor() const
     return m_timeDivisor;
 }
 
-QMap<QString, QStringList> LogdataStorage::getFmtValues() const
+QMap<QString, QStringList> LogdataStorage::getFmtValues(const bool filterStringValues) const
```

Comment:
you are passing a copy of fundamental type, it doesn't need to be declared const